JedisMaker.java

```diff
@@ -0,0 +1,131 @@
+package com.flatironschool.javacs;
+
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.net.URLDecoder;
+
+import redis.clients.jedis.Jedis;
+
+
+public class JedisMaker {
+
+	/**
+	 * Make a Jedis object and authenticate it.
+	 *
+	 * @return
+	 * @throws IOException
+	 */
+	public static Jedis make() throws IOException {
+
+
+		// assemble the file name
+		String slash = File.separator;
+		String filename = "resources" + slash + "redis_url.txt";
+		URL fileURL = JedisMaker.class.getClassLoader().getResource(filename);
+                String filepath = URLDecoder.decode(fileURL.getFile(), "UTF-8");
+
+                StringBuilder sb = new StringBuilder();
+		BufferedReader br;
+		try {
+                    br = new BufferedReader(new FileReader(filepath));
+		} catch (FileNotFoundException e1) {
+                    System.out.println("File not found: " + filename);
+                    printInstructions();
+                    return null;
```

Comment:
In several places here you return null in case of failure, but none of the call sites of make() actually check for it. Either update the call sites to check for it, or alternatively consider an exception in case of failure.